cmd/boot-script-service/routers.go

```diff
@@ -36,37 +36,86 @@
 package main
 
 import (
+	"context"
 	"fmt"
-	base "github.com/Cray-HPE/hms-base"
 	"net/http"
```

Comment:
Is `net/http` still needed if we are using `chi` instead?

Reply:
Yes, for the handler function definitions.

Reply:
Understood.